src/feature/flow_chart_panel/components/node-edit-menu/ParamField.tsx

```diff
@@ -60,6 +62,10 @@ const ParamField = ({
       );
     case "select":
       return <Select onChange={handleChange} data={options!} value={value} />;
+    case "node_reference":
+      return (
+        <Select onChange={handleChange} data={otherNodeIds!} value={value} />
```

Comment:
We can use nullish coalescing `??` here to avoid eslint error. e.g `otherNodeIds ?? []`